L1Trigger/L1THGCalUtilities/python/stage2FileWriter_cff.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+Stage2FileWriter = cms.EDAnalyzer('Stage2FileWriter',
```

Comment:
Is this implemented?

Reply:
Apologies, slipped through after switching branches.  It's for the pattern file generation for the interface tests.  Removed (and for next comment).